docs/changehistory/Roadmap.md

```diff
@@ -21,123 +21,129 @@ Volunteers for help on any or all of these projects are welcome. In particular,
+- [X] Expand base map support to include additional sources (AzureMaps, MapBox etc.) or solid color. Transparency of base map can be controlled independently of layers.
+- [X] Add support for display of raster map layers connected to live WMS, WMTS, AzureMaps, MapBox, or ArcGIS REST API servers. Map layer visibility and transparency can be controlled independently. Layers can be attached either as background below BIM geometry or overlays above BIM geometry.
 
 ### Microsoft Teams integration
 
-- Provide api for hosting iTwin.js applications directly within Microsoft Teams.
+- [ ] Provide api for hosting iTwin.js applications directly within Microsoft Teams.
 
 ### ET/IT/OT Integration
 
-- Provide working examples and templates for interfacing between engineering content in iModels, enterprise systems, and realtime IOT sensors, cameras, controllers, devices, and processors.
-- Create tools for augmenting data synchronized with connectors from engineering design tools with IOT-link elements.
-- Enhance BIS schemas for common IOT patterns and query engines.
+- [ ] Provide working examples and templates for interfacing between engineering content in iModels, enterprise systems, and realtime IOT sensors, cameras, controllers, devices, and processors.
+- [ ] Create tools for augmenting data synchronized with connectors from engineering design tools with IOT-link elements.
+- [ ] Enhance BIS schemas for common IOT patterns and query engines.
 
 ### Agent Deployment
 
-- Provide working examples and templates of agents deployed via Kubernetes.
-- Perform operations on iModels that are triggered by relevant events from iModelHub.
+- [ ] Provide working examples and templates of agents deployed via Kubernetes.
+- [ ] Perform operations on iModels that are triggered by relevant events from iModelHub.
 
 ### iTwin Viewer
 
-- Create fully functional iTwin web viewing application platform, deployable on any cloud infrastructure. The Bentley iTwin Design Review product will be built from this application.
-- Provide necessary infrastructure to host Extensions within any iTwin Viewer-based solution.
+- [ ] Create fully functional iTwin web viewing application platform, deployable on any cloud infrastructure. The Bentley iTwin Design Review product will be built from this application.
+- [ ] Provide necessary infrastructure to host Extensions within any iTwin Viewer-based solution.
 
 ### Point Clouds
 
-- Support visualization of time-based differencing of large scale point clouds.
+- [ ] Support visualization of time-based differencing of large scale point clouds.
 
 ### Reality Data Sources
 
-- Implement a grid of linear transformations for large reality data.
+- [ ] Implement a grid of linear transformations for large reality data.
```

Comment:
@MarcBedard8 @rbois-bentley I don't know what this means - do you?

Reply:
I beleive this is related to changes @RBBentley did. When a Reality Data is large instead of using a single transform for the whole Reality Data then a grid of multiple transforms is created that implement slightly different transforms across the Reality Data spawn. This enables to take into account that properly geolocated iModels are not linear transformed with regard to the 3D ECEF space. I beleive this has been completed a while ago.